algorithms/src/crh/bowe_hopwood_pedersen.rs

```diff
@@ -110,9 +114,8 @@ impl<G: Group, S: PedersenSize> CRH for BoweHopwoodPedersenCRH<G, S> {
                 S::NUM_WINDOWS,
             ));
         }
-        // we cant use these in array sizes since they are from a trait (and cant be refered to at const time)
-        assert!(S::WINDOW_SIZE <= 256);
-        assert!(S::NUM_WINDOWS <= 296);
+        assert!(S::WINDOW_SIZE <= MAX_WINDOW_SIZE);
+        assert!(S::NUM_WINDOWS <= MAX_NUM_WINDOWS);
 
         // overzealous but stack allocation
         let mut buffer = [0u8; 296 * 256 / 8 + BOWE_HOPWOOD_CHUNK_SIZE + 1];
```

Comment:
I believe you should incorporate usage of `MAX_WINDOW_SIZE` and `MAX_NUM_WINDOWS` here

Reply:
That would make sense lol